src/dynamic_boundary_conditions/tide/sea_level_rise_data.py

```diff
@@ -27,7 +20,9 @@
 log = logging.getLogger(__name__)
 
 
-def download_slr_data_files_from_takiwa(slr_data_dir: pathlib.Path) -> None:
+def download_slr_data_files_from_takiwa(
+    slr_data_dir: pathlib.Path
```

Comment:
I recommend reading the files directly into memory instead of downloading them to your local drive and using up space. You can then perform the necessary processing from there.